compiler/luci-interpreter/src/core/KernelParams.h

```diff
@@ -89,6 +89,7 @@ struct DivParams
 struct FullyConnectedParams
 {
   Activation activation;
+  bool keep_num_dims = false;
```

Comment:
(not for this PR) For this file, our convention is not to set initial values. But I think it would be better to set them for SAM score.